script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -86,23 +86,23 @@ I mean, even with a gag order, there's going to be leaks, right?"
 [sha:5ba91d11f604e1b69d264f590de14d02c159f498]{
 -- Page 791, Offset 13002.
 -- 「気にすんな。ニホン人は仕事熱心なんだろ？　秘密厳守っていう組織の方針には従わなきゃあな。
-"Don't sweat it. Japanese people are passionate about their work, no? They'll have to keep the secret if that is what their organization's policy is.
+"Don't sweat it. Japanese people are passionate about their work, no? If their organization's policy states it, they'll have no choice but to obey.
 }
 [sha:9eddb6040a007371b45faa774a25d0af87efa18f]{
 -- Page 791, Offset 13003.
 -- よしんば話が漏れても噂止まりだ。事故現場に死体があるのは不思議じゃねえんだしな」
-Even if it leaks, it'll be nothing more than a rumor. Besides, there's nothing incredible to having dead bodies at the scene of an accident."
+Even if it leaks, it'll be nothing more than a rumor. Besides, there's nothing incredible about dead bodies being present at the scene of an accident."
 }
 [sha:48e47085a1a91a3e70c500924664bbc64955c91a]{
 -- Page 792, Offset 13004.
 -- 「そりゃ不思議じゃねぇですけど、不自然ではありますよぉ？　どこにこんだけの死体があったんだって話ですし」
-// Recheck here
-"Well, maybe it's not incredible, but isn't there something strange about that? Like, where were all the dead bodies?"
+// Recheck here not too sure either
+"Well, maybe it's not incredible, but isn't there something strange about it? Like, where did all these bodies come from?"
```

Comment:
Well, it might not be %{i}incredible%{/i}, but isn't there something unnatural about it? Like, where'd all these bodies come from?